easybuild/easyconfigs/b/Blitz++/Blitz++-1.0.2-GCCcore-10.2.0.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'CMakeMake'
+
+name = 'Blitz++'
+version = '1.0.2'
+
+homepage = 'https://github.com/blitzpp/blitz'
+
+description = """
+ Blitz++ is a (LGPLv3+) licensed meta-template library for array manipulation
+ in C++ with a speed comparable to Fortran implementations, while preserving an
+ object-oriented interface
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+toolchainopts = {'pic': True}
+
+source_urls = [('https://github.com/blitzpp/blitz/archive/')]
+sources = ['%(version)s.tar.gz']
+checksums = ['500db9c3b2617e1f03d0e548977aec10d36811ba1c43bb5ef250c0e3853ae1c2']
+
+builddependencies = [('CMake', '3.18.4'), ('binutils', '2.35')]
+
+separate_build_dir = True
+
+postinstallcmds = ['mkdir %(installdir)s/lib64/blitz']
```

Comment:
I don't see how this is necessary? It's post install, so, it just creates an empty directory?

Reply:
I suspect this left over from some other tool I was trying to build that looked for this folder and complained when it didn't exist.  I've removed it now.